tests/bson/bson-utcdatetime-004.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+BSON BSON\UTCDateTime constructor defaults to current time
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$tests = [
+    new MongoDB\BSON\UTCDateTime,
+    new MongoDB\BSON\UTCDateTime(null),
```

Comment:
I think "null" should just be coerced to ``0``, and not it being empty. Hence, the milliseconds should end up being 0 (e.g., the Epoch)